metadata.rb

```diff
@@ -6,5 +6,5 @@
 long_description 'Installs/Configures deploy-user'
 version '1.4.2'
 
-depends 'ssh_known_hosts', '~> 0.2'
+depends 'ssh_known_hosts', '~> 4.0'
```

Comment:
in fact, forcing to version 4.x
it will fetch the Supermarked cookbook https://supermarket.chef.io/cookbooks/ssh_known_hosts and it seems to work fine
Couldn't we just use that as a dependency?